kart/sqlalchemy/create_engine.py

```diff
@@ -18,6 +18,21 @@
 L = logging.getLogger("kart.sqlalchemy.create_engine")
 
 
+def sqlite_engine(path):
+    """
+    An engine for non-spatial, non-GPKG sqlite databases.
+    """
+
+    def _on_connect(pysqlite_conn, connection_record):
+        pysqlite_conn.isolation_level = None
+        dbcur = pysqlite_conn.cursor()
+        dbcur.execute("PRAGMA foreign_keys = ON;")
```

Comment:
need to set journal mode to WAL on DB creation otherwise multiple processes will clash.